<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelPreviewDialog</class>
 <widget class="QDialog" name="ModelPreviewDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PRISM model preview</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="modelTab">
      <attribute name="title">
       <string>Model</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="modelText">
         <property name="font">
          <font>
           <family>Courier 10 Pitch</family>
          </font>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="propertiesTab">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="pctlRadioButton">
            <property name="text">
             <string>Probabilistic Computation Tree Logic (PCTL)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ctlRadioButton">
            <property name="text">
             <string>Computation Tree Logic (CTL)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="propertiesText">
         <property name="font">
          <font>
           <family>Courier 10 Pitch</family>
          </font>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModelPreviewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>693</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModelPreviewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>693</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pctlRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>ModelPreviewDialog</receiver>
   <slot>propTypeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>977</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ctlRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>ModelPreviewDialog</receiver>
   <slot>propTypeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>979</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>propTypeChanged()</slot>
 </slots>
</ui>
